### PR TITLE
Fix accessiblenamespace scale

### PR DESCRIPTION
**Pull request by t00f**
In the process of fixing the accessible namespaces scaling issues, we are removing all properties except the name.

The output result would be something like:

```
apoctl api list accessiblenamespaces
[
  {
    "name": "/tenant/cloudaccount1"
  },
  {
    "name": "/tenant/cloudaccount2"
  }
]
```

### CODE REVIEW

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "gaia",
    "pr-id": "639",
    "commit-sha": "4662fbdd4e0c92aa5bb6d8da77afbf27fbeb687b",
    "pipeline": "master"
  },
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "backend",
    "pr-id": "1142",
    "commit-sha": "2f089d08d5d7385d7e03f1e0c9e25c057e0b290c",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "gaia",
    "pr-id": "639",
    "commit-sha": "6643998345a8f41e0c3676323054b7d5eaeb03da",
    "pipeline": "master"
  },
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "backend",
    "pr-id": "1142",
    "commit-sha": "bd60c8bc2f9fb2bac64fd804ec23923ec18dde19",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "gaia",
    "pr-id": "639",
    "commit-sha": "6643998345a8f41e0c3676323054b7d5eaeb03da",
    "pipeline": "master"
  },
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "backend",
    "pr-id": "1142",
    "commit-sha": "fe7f46585a3127182a65b17b192b4e9c5927907b",
    "pipeline": "master"
  }
]
```

File: specs/accessiblenamespace.spec [hunk 1]
```diff
@@ -10,8 +10,6 @@ model:
     user.
   aliases:
   - accns
-  extends:
-  - '@namespaced'
```
Comment (primalmotion) [inline specs/accessiblenamespace.spec:14]: you should not remove this. instead populate it with the ns of the request

**Review by primalmotion — CHANGES_REQUESTED** [review]

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "gaia",
    "pr-id": "639",
    "commit-sha": "6643998345a8f41e0c3676323054b7d5eaeb03da",
    "pipeline": "master"
  },
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "backend",
    "pr-id": "1142",
    "commit-sha": "dc4f20883770bdc2c9c5315b50d30b79795fee08",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "backend",
    "pr-id": "1142",
    "commit-sha": "1356b595b5595131fecb1d2647b97c6e599f470b",
    "pipeline": "master"
  },
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "gaia",
    "pr-id": "639",
    "commit-sha": "6a96101c61d19ee8d32627536867fb6b4324b5e9",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "gaia",
    "pr-id": "639",
    "commit-sha": "6a96101c61d19ee8d32627536867fb6b4324b5e9",
    "pipeline": "master"
  },
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "backend",
    "pr-id": "1142",
    "commit-sha": "9999be1d9a1630544881af8ffb7fe34ab3cdb26d",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "gaia",
    "pr-id": "639",
    "commit-sha": "6a96101c61d19ee8d32627536867fb6b4324b5e9",
    "pipeline": "master"
  },
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "backend",
    "pr-id": "1142",
    "commit-sha": "f05b646779e16233a007851ca387260135aec893",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "backend",
    "pr-id": "1142",
    "commit-sha": "f95f6eb173e99675de319454db24140bad793c85",
    "pipeline": "master"
  },
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "gaia",
    "pr-id": "639",
    "commit-sha": "6a96101c61d19ee8d32627536867fb6b4324b5e9",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "gaia",
    "pr-id": "639",
    "commit-sha": "6a96101c61d19ee8d32627536867fb6b4324b5e9",
    "pipeline": "master"
  },
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "backend",
    "pr-id": "1142",
    "commit-sha": "ec41dab3008552e45fb9cdccf9249441c7c80016",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "gaia",
    "pr-id": "639",
    "commit-sha": "24bd9861153b501847473712a7c5d827137e28b4",
    "pipeline": "master"
  },
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "backend",
    "pr-id": "1142",
    "commit-sha": "ec41dab3008552e45fb9cdccf9249441c7c80016",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "gaia",
    "pr-id": "639",
    "commit-sha": "24bd9861153b501847473712a7c5d827137e28b4",
    "pipeline": "master"
  },
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "backend",
    "pr-id": "1142",
    "commit-sha": "89c03dfc430c66f7769670026418a7c19992c57e",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "gaia",
    "pr-id": "639",
    "commit-sha": "24bd9861153b501847473712a7c5d827137e28b4",
    "pipeline": "master"
  },
  {
    "project": "fix_accessiblenamespace_scale",
    "component": "backend",
    "pr-id": "1142",
    "commit-sha": "4c6b830e94214fdbea6723c2819f065f0fb5a4dc",
    "pipeline": "master"
  }
]
```

**Pull request merged**